docker-compose.yml

```diff
@@ -73,8 +74,10 @@ services:
   superset-node:
     image: node:12
     container_name: superset_node
-    command: ["bash", "-c", "cd /app/superset-frontend && npm install -f --no-optional --global webpack webpack-cli && npm install -f --no-optional && npm run dev"]
+    command: ["/app/docker-frontend.sh"]
     env_file: docker/.env
+    environment:
```

Comment:
If running the dev server, you probably need to expose port 3000, right?

Reply:
Sorry for huge delay with this PR. 

I tested this again and my observations are:
- it was not needed to switch to dev-server from dev - > so I removed ENV variable logic
- Exposing any different port than 8088 also was not needed - > when we point in the browser localhost:8088 after all webpack work is done - preview in the terminal is something like:
```superset_node            | clean-webpack-plugin: dry ../superset/static/assets/vendors.6b0d30c5.chunk.js
superset_node            |    10353 modules
superset_node            | No type errors found
superset_node            | Version: typescript 4.0.3
superset_node            | Time: 88356ms```
then after submitting any change to UI -> this change is reflected